moped-editor/src/views/projects/projectView/ProjectMilestones/DataGridDateFieldEdit.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { MobileDatePicker } from "@mui/x-date-pickers";
+import { parseISO, format } from "date-fns";
+import { useGridApiContext } from "@mui/x-data-grid-pro";
+import theme from "src/theme";
+
+/**
+ * DataGridDateFieldEdit - renders a Date type Calendar select for use in DataGrid
+ * @param {string} value - the current value
+ * @param {Integer} id - Data Grid row id
+ * @param {String} field - name of field
+ * @param {Boolean} hasFocus - does this field have focus
+ * @return {JSX.Element}
+ * @constructor
+ */
+const DataGridDateFieldEdit = ({ value, hasFocus, id, field, ...props }) => {
+  const apiRef = useGridApiContext();
+  const ref = React.useRef(null);
+
+  React.useEffect(() => {
+    if (hasFocus) {
+      ref.current.focus();
+    }
+  }, [hasFocus]);
+
+  const handleChange = (newValue) => {
+    const newDate = newValue ? format(newValue, "yyyy-MM-dd") : null;
+    apiRef.current.setEditCellValue({
+      id,
+      field,
+      value: newDate,
+    });
+  };
+
+  return (
+    <MobileDatePicker
+      sx={{ padding: theme.spacing(1) }}
```

Comment:
Thanks for spacing these out some more - could you update this to `marginX` so that there isn't extra space added in Y direction? Thank you! 🙏